plugins/woocommerce/includes/class-wc-structured-data.php

```diff
@@ -577,4 +577,31 @@ public function generate_order_data( $order, $sent_to_admin = false, $plain_text
 
 		$this->set_data( apply_filters( 'woocommerce_structured_data_order', $markup, $sent_to_admin, $order ), true );
 	}
+
+	/**
+	 * Check if a GTIN is valid.
+	 * A GTIN is valid if:
+	 * Is a positive numeric string with a length of 8,12,13 or 14.
```

Comment:
Should we change this comment as well now:

```suggestion
	 * A valid GTIN is a string containing 8,12,13 or 14 digits.
```